src/lib/y2network/autoyast_routing.rb

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/routing_helpers"
+
+Yast.import "Lan"
+
+module Y2Network
+  # This class is responsible of importing and exporting AutoYast routing
+  # configuration
+  class AutoyastRouting
```

Comment:
@jreidinger @imobachgs @mchf 

`RoutingAutoyast` probably should be better, although what about using another adding an extra namespace level here? Something like `Y2Network::Routing::Autoyast` ...`Y2Network::Routing::Config` `Y2Network::Routing::Route`, I know it is not a loved pattern so ... what do you think?

Reply:
Well, I am not so big fun of that deep namespaces, as it makes names resolution quite tricky. Consider this example:
```
# we already have this classes
Y2Network::Route
Y2Network::Dialogs::Route

# and consider this code:
module Y2Network
  module Dialogs
    class A
       Route.new # which class it will be? It depends on require and if you forget to require dialogs/route then it will be backend Route.
```

So to sum it, I prefer less namespaces unless it is really logical separation.